src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -60,10 +64,9 @@ public void drawRoute(final Graphics2D graphics, final RouteDescription routeDes
     final int jointsize = 10;
```

Comment:
Please move the declaration of `jointsize` (line 64) to be before line 78, where it is first used.
Also consider removing the `jointsize` parameter from  `drawJoints` on line 78:
` drawJoints(graphics, points, xOffset, yOffset, jointsize, scale);`, and instead `drawJoints` would use the `jointsize` constant directly.